### DISCUSSION TITLE
Resource not accessible by personal access token with PR

### DISCUSSION THREAD

**Discussion by seehi** [community/community#123362] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

According the [create-a-pull-request](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#create-a-pull-request), create `Fine-grained access tokens` with `pull_requests:write`, but encounter `Resource not accessible by personal access token`.

Does API can create a pr to FORKOWNER? Sincerely ask for your help.

**Comment by seehi** [comment]
# Input
curl -v -L \
  -X POST \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer MY_TOKEN" \
  https://api.github.com/repos/FORKOWNER/REPO/pulls \
  -d '{"title":"Test for pr","body":"Please pull these awesome changes in!","head":"MY_USERNAME:NEW_BRANCH","base":"main"}'
 # Output
 {
  "message": "Resource not accessible by personal access token",
  "documentation_url": "https://docs.github.com/rest/pulls/pulls#create-a-pull-request"
}

**Comment by seehi** [comment]
Resolved，using `classic token` instead of `fine grained token`.

**Reply by jakub-trzebiatowski** [reply]
Well, this doesn't resolve the problem _really_, as classic tokens are extremely powerful (too powerful for many use cases). PATs should be able to create PRs...

**Reply by RodrickSia** [reply]
Do you have any idea what is going on with this api? The docs said a token with administration permission of read and write should be enough, but using it to request just have permission denied.
